### ISSUE TITLE
Understanding metrics

### ISSUE THREAD

**Issue by abaybektursun** [OFA-Sys/OFA#206]
I am trying to understand the metrics being logged to weights and biases.
Am understanding these correctly?
**`wps` - words per second?**
**`wbp` - Seems to be related to `wps`, but what is it?**
```python
metrics.log_scalar("wpb", ntokens, priority=180, round=1)
metrics.log_speed("wps", ntokens, priority=90, round=1)
```
`clip` - I believe CLIP loss using OpenAI's CLIP
**`lossv1 and v2` - What are these?:**
```python
loss_v1, sample_size_v1, logging_output_v1 = self.forward(model, sample[0], update_num, reduce)
loss_v2, sample_size_v2, logging_output_v2 = self.forward(model, sample[1], update_num, reduce)
loss = loss_v1 / sample_size_v1 + loss_v2 / sample_size_v2
```
`loss` - Loss
**`ups` - What is this?**
`nll_loss` - The negative log likelihood loss
`ppl` - Perplexity
`gnorm` - Gradient Norm
`lr` - learning rate
`wall` - wall-clock time?
`gb_free`: free memory
```python
gb_free = self.cuda_env.total_memory_in_GB - gb_used
```
`bsz` - batch size

**Comment by JustinLin610** [comment]
loss v1 and v2 are connected with the pretraining. As OFA is pretrained with data of multimodality and unimodality (plain text / image), v1 and v2 correspond to them respectively. clip should be related to gradient clipping instead of OpenAI CLIP. For the others you can refer to https://github.com/facebookresearch/fairseq/issues/2201
